xAPI.md

```diff
@@ -4539,6 +4540,13 @@ property MUST occur only once.
 * An LRS MUST NOT reject requests based on their version header as would otherwise be 
 required by <a href="#versioning"/>Versioning</a>.
 
+<a name="encoding"/> 
+
+### 2.3 Encoding
+
+###### Requirement
+* All strings MUST be encoded and interpreted as UTF-8. 
```

Comment:
I know we're just re-adding, but some ideas on making this more precise:

* All JSON MUST be encoded and interpreted as UTF-8.
* All strings described in the specification for use outside of JSON documents, such as request parameters, MUST be encoded and interpreted as UTF-8

The reason for the split is, from a technical perspective, JSON encoding doesn't apply to the individual strings within JSON, but to the JSON representation as a whole